src/emulator/mem/include/mem/mem.h

```diff
@@ -37,7 +38,8 @@ struct MemState {
     Allocated allocated_pages;
     std::mutex generation_mutex;
     GenerationNames generation_names;
-    std::map<Address, uint32_t> breakpoints;
+
+    std::unique_ptr<std::array<uint8_t *, 1 << 20>> pages_cpu;
```

Comment:
Could put the `MB` prototype above the struct and
```suggestion
    std::unique_ptr<std::array<uint8_t *, MB(1)>> pages_cpu;
```